lib/signs_ui/config/expiration.ex

```diff
@@ -59,6 +59,59 @@ defmodule SignsUi.Config.Expiration do
     |> Enum.into(%{})
   end
 
+  @spec expire_signs_via_alert(list(SignsUi.Config.State.t()), MapSet.t()) ::
+          list(SignsUi.Config.State.t())
+  def expire_signs_via_alert(sign_states, alert_ids) do
+    expired_sign_states =
+      get_expired_sign_states(
+        sign_states,
+        alert_ids,
+        []
+      )
+
+    expired_sign_states
+  end
+
+  @spec get_expired_sign_states(
+          list(SignsUi.Config.State.t()),
+          MapSet.t(),
+          list(SignsUi.Config.State.t())
+        ) :: list(SignsUi.Config.State.t())
+  def get_expired_sign_states(remaining_sign_states, alert_ids, expired_sign_states) do
```

Comment:
This can probably be a private function, since it's an implementation detail of `expire_signs_via_alert`.